0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -33,108 +31,90 @@ The sending and receiving ledger are different ledgers. Otherwise, there should
 
 ## Get Quote
 
-Quotes are requested with `plugin.sendMessage(message)` and returned via `plugin.on("incoming_message")` (see the Ledger Plugin Interface for more details).
-The message formats described here are the `data` properties of an `OutgoingMessage`.
+Quotes are requested with `plugin.sendRequest(message)`, which returns the quote response (see the Ledger Plugin Interface for more details).
 
-### Request Message Format
+A quote request's `ilp` property must be an `IlqpLiquidityRequest`, `IlqpBySourceRequest`, or `IlqpByDestinationRequest`. The response's `ilp` property must be an `IlqpLiquidityResponse`, `IlqpBySourceResponse`, or `IlqpByDestinationResponse` respectively. If an error occurs during quoting, `ilp` will be a [`IlpError`](../0003-interledger-protocol/0003-interledger-protocol.md#ilp-error-format) instead.
```

Comment:
[Message#ilp](https://github.com/interledger/rfcs/blob/e4dab0259283a500f923f92ca4f50b18509cdc44/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#messageilp) could refer to any ILP packet. Since this is the ILQP spec, I only mention the ones applicable to quoting.